src/ai/backend/manager/models/kernel.py

```diff
@@ -968,6 +976,16 @@ async def resolve_commit_status(self, info: graphene.ResolveInfo) -> Optional[st
         commit_status = await graph_ctx.registry.get_commit_status(self.id, self.access_key)
         return commit_status["status"]
 
+    async def resolve_abusing_reports(
+        self, info: graphene.ResolveInfo
+    ) -> Iterable[Mapping[str, Any]]:
```

Comment:
abusing report can be `None`.
```suggestion
    ) -> Iterable[Optional[Mapping[str, Any]]]:
```